animareflection/ui
# Button Variants

## Description

##### Task link: https://trello.com/c/mzEH1qkr/139-button-variants

Added custom `primary`, `secondary`, `ghost`, and `round` variants for Core UI `Button` component. Also relocated the button `recipe` to a dedicated file to match patterns of other components.

## Test Steps

1) Verify that `variant` prop can be passed to `Button` component to change design pattern of the component in a downstream app.
2) Verify that controls for said `variant` prop work appropriately in said component
